assets/prefabs/Equipments/plazmaster.prefab

```diff
@@ -1,7 +1,7 @@
 {
   "parent": "engine:iconItem",
   "DisplayName": {
-    "name": "PlazMaster 3000"
+    "name": "plazmaster"

```

Comment:
Any reason for the name change? Seems like "plazmaster" is less player readable than "Plazmaster 3000"
I'd at least like to see it capitalised.

EDIT: See comment below about upgrade command

Reply:
The name is set this way because of its specific usage in `upgrade` console command. `.getParentPrefab().getName()` results in an NPE currently, hence I am keeping it this way